rholang/src/main/scala/coop/rchain/rholang/interpreter/PrettyPrinter.scala

```diff
@@ -52,6 +52,8 @@ final case class PrettyPrinter(
   private def buildStringM(u: GUnforgeable): Coeval[String] = Coeval.defer {
     u.unfInstance match {
       case GPrivateBody(p) => pure("Unforgeable(0x" + Base16.encode(p.id.toByteArray) + ")")
+      case GDeployIdBody(id) =>
```

Comment:
Sorry for forgetting about that.

Reply:
No worries, so did I during the review :) Rholang tests for syntax elements should really be structured to catch those, but you know what they say: it is what it is...